<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UBTActionWidget</class>
 <widget class="QWidget" name="UBTActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>158</width>
    <height>177</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>158</width>
    <height>177</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>158</width>
    <height>177</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="thumbnailLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>158</width>
     <height>108</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>158</width>
     <height>108</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>158</width>
     <height>108</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadBtn">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>84</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>34</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>34</width>
     <height>24</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>138</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>138</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>138</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="statusLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>138</width>
     <height>12</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>138</width>
     <height>12</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>138</width>
     <height>12</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="actionCoverLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>158</width>
     <height>108</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="redownloadBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>78</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>78</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>78</width>
     <height>28</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>re-download</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
